enerdata/contracts/tariff.py

```diff
@@ -394,16 +394,32 @@ def __init__(self):
 
 class T31A(T30A):
     """
-    3.1A Tariff
+    3.1A Tariff patched with 3.1ALB
     """
-    def __init__(self):
+    def __init__(self, kva=None):
         super(T31A, self).__init__()
         self.code = '3.1A'
         self.cof = 'C'
         self.min_power = 1
         self.max_power = 450
+        self.losses = 0.04
         self.type = 'AT'
         self.require_powers_above_min_power = False
+        if kva:
+            if not isinstance(kva, int or float):
+                raise ValueError('kva must be an enter value')
+            self.LB = True
+            self.kva = kva
+        else:
+            self.LB = False
```

Comment:
I don't like the properties in upper cases, in a standard it denotes CONSTANT_VALUES and it isn't a constant it's a property. What about `low_voltage_measure` or something

Reply:
I think same!

Reply:
Done via: https://github.com/gisce/enerdata/pull/112/commits/5a4796099ab50f0630fdc415c7436f3bcace0191